src/app/app.css

```diff
@@ -77,3 +77,94 @@ html, body, #root {
 .recording-table-container {
   height: 32rem;
 }
+
+.automated-analysis-grid-item {
+  padding: 0.25em;
+}
+
+.automated-analysis-score-filter-stack-item {
+  padding-left: 1.3em;
+  padding-right: 1.3em;
+}
+
+.automated-analysis-score-filter-slider .pf-c-slider__main {
+  margin-left: 1em;
+  --pf-c-slider__step--m-active__slider-tick--BackgroundColor: var(--pf-global--BorderColor--200);
+  --pf-c-slider__rail-track--before--fill--BackgroundColor: var(--pf-global--BorderColor--100);
+}
+
+.automated-analysis-score-filter-slider-ok .pf-c-slider__main {
+  --pf-c-slider__rail-track--before--base--BackgroundColor: var(--pf-global--success-color--100);
+  --pf-c-slider__step-tick--BackgroundColor: var(--pf-global--success-color--200);
+}
+
+.automated-analysis-score-filter-slider-warning .pf-c-slider__main {
+  --pf-c-slider__rail-track--before--base--BackgroundColor: var(--pf-global--warning-color--100);
+  --pf-c-slider__step-tick--BackgroundColor: var(--pf-global--warning-color--200);
+}
+
+.automated-analysis-score-filter-slider-critical .pf-c-slider__main {
+  --pf-c-slider__rail-track--before--base--BackgroundColor: var(--pf-global--danger-color--100);
+  --pf-c-slider__step-tick--BackgroundColor: var(--pf-global--danger-color--300);
+}
+
+.automated-analysis-score-filter-slider .pf-c-slider__value .pf-c-form-control {
+  width: 4em;
```

Comment:
I think mayube 5em might be a better choice since it allows enough space for 4 numbers  (including leading 0 if any)

![width-input](https://user-images.githubusercontent.com/68053619/204720685-edc1c5af-ca0d-4e73-ba8b-586f012ba009.png)